CODEOWNERS

```diff
@@ -0,0 +1,11 @@
+*                                   @chxchx @cjwagner @fejta
+/prow/config/                       @chxchx @cjwagner @fejta @howardjohn @sdake @geeknoid
+/prow/cluster/jobs/                 @howardjohn @sdake @geeknoid @duderino @linsun
+/prow/cluster/jobs/istio/bots/      @howardjohn @sdake @geeknoid @duderino @linsun @ozevren
+/prow/cluster/jobs/istio/community/ @howardjohn @sdake @geeknoid @duderino @linsun
+/prow/cluster/jobs/istio/installer/ @howardjohn @sdake @geeknoid @duderino @linsun @costinm
+/prow/cluster/jobs/istio/istio.io/  @howardjohn @sdake @geeknoid @duderino @linsun @frankbu
+/prow/cluster/jobs/istio/istio/     @howardjohn @sdake @geeknoid @duderino @linsun @costinm @rshriram
+/prow/cluster/jobs/istio/operator/  @howardjohn @sdake @geeknoid @duderino @linsun @ayj @costinm @Laci21 @martonsereg @matyix @ostromart @rcernich @waynz0r
+/prow/cluster/jobs/istio/pkg/       @howardjohn @sdake @geeknoid @duderino @linsun @ozevren
+/prow/cluster/jobs/istio/tools/     @howardjohn @sdake @geeknoid @duderino @linsun @mandarjog
```

Comment:
Could you add me to `/prow/cluster/jobs/istio/proxy/`? Thanks!

Reply:
Done.